File: sdk/samples/WCFSamples/TechnologySamples/Scenario/DataBinding/WPFDataBinding/CS/service/IAlbumService.cs
//  Copyright (c) Microsoft Corporation. All rights reserved.

using System.ServiceModel;

namespace Microsoft.ServiceModel.Samples
{
	[ServiceContractAttribute(Namespace = "http://Microsoft.ServiceModel.Samples")]
    interface IAlbumService
    {
        [OperationContract]
        Album[] GetAlbumList();

		[OperationContract]
        void AddAlbum(string title);
    }
}

File: sdk/samples/WCFSamples/TechnologySamples/Scenario/DataBinding/WPFDataBinding/VB/service/IAlbumService.vb
' Copyright (c) Microsoft Corporation. All rights reserved.

Imports System.ServiceModel

Namespace Microsoft.ServiceModel.Samples

    <ServiceContractAttribute([Namespace]:="http://Microsoft.ServiceModel.Samples")> _
    Interface IAlbumService

        <OperationContract()> _
        Function GetAlbumList() As Album()
        <OperationContract()> _
        Sub AddAlbum(ByVal title As String)

    End Interface

End Namespace
